css/src/components/form/form.scss

```diff
@@ -61,7 +61,7 @@ $field-error-color: $danger !default;
 
 .required-indicator::after,
 ::part(required-indicator)::after {
-	color: $danger;
+	color: $danger-active;
```

Comment:
Not blocking, consider `$danger` here to match text with `.color-danger` for example.